src/types/Firestore.ts

```diff
@@ -66,5 +66,7 @@ export interface Firestore {
     venues?: Array<WithId<AnyVenue>>;
     events?: Array<WithId<VenueEvent>>;
     playaVenues?: Array<WithId<AnyVenue>>;
+    statsOnlineUsers: Array<WithId<User>>;
+    statsOpenVenues: Array<WithId<AnyVenue>>;
```

Comment:
make these optional. They are not defined before they are fetched from firebase.